features/networking/pod.feature

```diff
@@ -480,3 +480,37 @@ Feature: Pod related networking scenarios
     | ip route |
   Then the output should contain:
     | <%= cb.pod_cidr %> dev <%= cb.tunnel_inf_name %> |
+    
+  # @author anusaxen@redhat.com
+  # @case_id OCP-26373
+  @admin
+  @destructive  
+  Scenario: Pod readiness check for OVN		
+  Given the env is using "OVNKubernetes" networkType
+  And OVN is functional on the cluster
+  Given I switch to cluster admin pseudo user
+  And I use the "openshift-ovn-kubernetes" project
+  And a pod is present with labels:
+    | l | app=ovnkube-node |
+  #Removing CNI config file from container to check readiness probe functionality
+  When I run the :exec client command with:
+    | pod              | <%= pod.name %>                       |
+    | c                | ovnkube-node                          |
+    | oc_opts_end      |                                       |
+    | exec_command     | rm                                    |
+    | exec_command_arg | /etc/cni/net.d/10-ovn-kubernetes.conf |
+  Then the step should succeed
+  #Deleting ovnkube-pod will force CNO to rewrite the conf file and bring cluster back to normal after scenario
+  And I register clean-up steps:
+  """
+  admin ensure "<%= pod.name %>" pod is deleted from the "openshift-ovn-kubernetes" project
+  OVN is functional on the cluster
+  """
+  #Now make sure readiness probe checking above file will cause one of the two ovnkube-node containers to go down
+  Given 30 seconds have passed
+  When I run the :get admin command with:
+    | resource      | pod             |
+    | resource_name | <%= pod.name %> |
+  Then the step should succeed
+  And the output should contain: 
+    | 1/2 |
```

Comment:
What is the `1/2` meaning?

Reply:
Peter, couldn't guess other way to check READY value. Like in example below. Any suggestions are welcome though
# oc get pod ovnkube-node-w9f9h
NAME                 READY   STATUS    RESTARTS   AGE
ovnkube-node-w9f9h   1/2     Running   0          3h10m

Reply:
I can parse json though for READY bu though above would be easy way

Reply:
json parsing for container status would make more sense..Fixed it.